CRITICAL - 2025-06-10 16:15:36 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "catalogoview.php"
[Method: GET, Route: catalogo]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('catalogoview.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('catalogoview', [], true)
 3 APPPATH\Controllers\Home.php(65): view('catalogoview')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->catalogo()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 16:18:38 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "catalogoview.php"
[Method: GET, Route: catalogo]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('catalogoview.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('catalogoview', [], true)
 3 APPPATH\Controllers\Home.php(65): view('catalogoview')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->Catalogo()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 16:19:33 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "catalogo_view.php"
[Method: GET, Route: catalogo]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('catalogo_view.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('catalogo_view', [], true)
 3 APPPATH\Controllers\Home.php(65): view('catalogo_view')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->Catalogo()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 16:19:35 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "catalogo_view.php"
[Method: GET, Route: catalogo]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('catalogo_view.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('catalogo_view', [], true)
 3 APPPATH\Controllers\Home.php(65): view('catalogo_view')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->Catalogo()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 17:04:45 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: productos/crear]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\ProductoController.php(20): CodeIgniter\HTTP\IncomingRequest->getJSON()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->crear()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-10 17:06:25 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Cojunto traje,camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => assets/img/modelo1.webp
    [estado] => activo
    [id_categoria] => 3
)

ERROR - 2025-06-10 17:15:05 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Cojunto traje,camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => assets/img/messiteamo.jpg
    [estado] => activo
    [id_categoria] => 20
)

ERROR - 2025-06-10 17:16:31 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Cojunto traje,camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => https://www.google.com/imgres?q=imagen%20de%20ropa&imgurl=https%3A%2F%2Femprendedores.es%2Fwp-content%2Fuploads%2FGestion-de-proveedores-1024x683.jpg&imgrefurl=https%3A%2F%2Femprendedores.es%2Fgestion%2Fcrear-una-empresa%2Fplan-de-negocio-tienda-de-ropa%2F&docid=ere77TnjnsRnfM&tbnid=rHodhHPSsc1y4M&vet=12ahUKEwitqMTOreeNAxW7uJUCHTm7HmkQM3oECHcQAA..i&w=1024&h=683&hcb=2&ved=2ahUKEwitqMTOreeNAxW7uJUCHTm7HmkQM3oECHcQAA
    [estado] => activo
    [id_categoria] => 20
)

ERROR - 2025-06-10 17:17:16 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Cojunto traje,camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRKMdnKglhSQU4WKuNwsTHdTf46gpLvYAzt0Q&s
    [estado] => activo
    [id_categoria] => 20
)

ERROR - 2025-06-10 17:19:37 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Conjunto traje, camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => img/modelo1.webp
    [estado] => activo
    [id_categoria] => 3
)

ERROR - 2025-06-10 17:24:05 --> Datos recibidos: Array
(
    [nombre] => Conjunto señor de la noche
    [descripcion] => Conjunto traje, camisa y corbata
    [precio] => 550
    [stock] => 3
    [talla] => M
    [color] => Azul
    [imagen] => img/modelo1.webp
    [estado] => activo
    [id_categoria] => 3
)

DEBUG - 2025-06-10 22:18:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-10 22:36:21 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proyecto_taller\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto_taller\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto_taller\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\proyecto_taller\app\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto_taller\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\proyecto_taller\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\proyecto_taller\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-06-10 22:36:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:36:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:36:21 --> [Caused by] mysqli_sql_exception: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-10 22:36:32 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proyecto_taller\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto_taller\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto_taller\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\proyecto_taller\app\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto_taller\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\proyecto_taller\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\proyecto_taller\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-06-10 22:36:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:36:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:36:32 --> [Caused by] mysqli_sql_exception: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` IS NULL
AND `usuarios`.`deleted_at` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\AuthController.php(17): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:44:13 --> ParseError: syntax error, unexpected token "if"
[Method: POST, Route: login]
in APPPATH\Controllers\AuthController.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\proyecto_taller\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 22:57:27 --> Error: Cannot use object of type stdClass as array
[Method: POST, Route: login]
in APPPATH\Controllers\AuthController.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-10 23:06:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-10 23:07:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-10 23:08:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-10 23:08:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-10 23:12:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-10 23:12:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-10 23:12:29 --> mysqli_sql_exception: Table 'costura_fina.carrito' doesn't exist in C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proyecto_taller\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto_taller\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto_taller\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\proyecto_taller\app\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\proyecto_taller\app\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
#8 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
#9 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
#10 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyecto_taller\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyecto_taller\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyecto_taller\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-10 23:12:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'costura_fina.carrito' doesn't exist
[Method: POST, Route: carrito/agregar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 23:12:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'costura_fina.carrito' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 23:12:29 --> [Caused by] mysqli_sql_exception: Table 'costura_fina.carrito' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-10 23:13:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-10 23:13:36 --> mysqli_sql_exception: Table 'costura_fina.carrito' doesn't exist in C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\proyecto_taller\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\proyecto_taller\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\proyecto_taller\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto_taller\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto_taller\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\proyecto_taller\app\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\proyecto_taller\app\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
#8 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
#9 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
#10 C:\xampp\htdocs\proyecto_taller\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyecto_taller\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyecto_taller\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyecto_taller\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-10 23:13:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'costura_fina.carrito' doesn't exist
[Method: POST, Route: carrito/agregar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 23:13:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'costura_fina.carrito' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-10 23:13:36 --> [Caused by] mysqli_sql_exception: Table 'costura_fina.carrito' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `carrito`
WHERE `id_usuario` = \'2\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `carrito`
WHERE `id_usuario` = :id_usuario:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\CarritoModel.php(17): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\CarritoController.php(47): App\Models\CarritoModel->obtenerCarritoUsuario('2')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CarritoController->agregarProducto()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CarritoController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
